conda_forge_tick/mamba_solver.py

```diff
@@ -280,7 +280,7 @@ def is_recipe_solvable(feedstock_dir):
             arch,
         )
         solvable = solvable and _solvable
-        errors.extend(_errors)
+        errors.extend(["f{cbc_fname}: {e}" for e in _errors])
```

Comment:
I already put a fix in for this. Can you merge master in and figure out which one we should keep. I think yours might be better.

Reply:
Also you need to extract the basename here. This is a full path.

Reply:
made a hybrid